docs/workload-development-kit/authentication-tutorial.md

```diff
@@ -42,7 +42,7 @@ Connect-AzureAD
 New-AzureADServicePrincipal -AppId e406a681-f3d4-42a8-90b6-c2b029497af1
 ```
 
-## Configure your application in Microsoft Entra ID
+## Configure your application in Microsoft Entra ID - Manual
```

Comment:
Hi @muliwin - we don't really use hyphens in section headers. We also don't use 'please' and try to use simple words.

I suggest:

## Configure your application in Microsoft Entra ID

To work with authentication, you need an application registered in Microsoft Azure. If you don't have an app registered, follow the instructions in this section to register one.

### Register an Azure app manually

To register an Azure app manually, follow the instructions in [Register an application](/entra/identity-platform/quickstart-register-app#register-an-application).

### Register an Azure app using a script

Use this PowerShell script to register an Azure app without using the Azure portal.

1. [Install the Azure CLI](/cli/azure/install-azure-cli).

2. Execute the [CreateDevAADApp script](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Authentication/CreateDevAADApp.ps1). Sign in with the credentials of the user account you're using for the Entra ID application.

3. When prompted, enter a name for your app's workload prefixed with *Org.* and your tenant ID. For example, *Org.Myworkload*.

Upon successful execution of the script, it'll output the details you need to configure your workload. The script also provides a direct URL to your application and an administrative consent URL for tenant app authorization.

#### Script example

This example shows how to use the *CreateDevAADApp.ps1* script to automate the app setup process. Replace the tenant ID in the example with your tenant ID.

To create an app named *myWorkloadApp* with the workload name *Org.Myworkload*, run the following command in PowerShell:

```powershell
powershell .\CreateDevAADApp.ps1 -applicationName "myWorkloadApp" -workloadName "Org.Myworkload" -tenantId "cb1f79ad-7930-43c0-8d84-c1bf8d15c8c7"
```